uwrt_mars_rover_blackfly_calibration/README.md

```diff
@@ -0,0 +1,15 @@
+# Brief
+This package can be used for calibrating the blackfly cameras. The calibration process consists of 2 nodes. The first node (broadcaster) broadcasts images taken from the blackfly to some topic. The 2nd node (cameracalibrator) uses these images to compute camera intrinsics.
+
+# Pre-req
+Make sure the [Spinnaker SDK](https://www.flir.ca/products/spinnaker-sdk/) is installed. As a sanity test, run the SpinView application.
```

Comment:
Maybe leave the exact commands for running spinview. (iirc its should add the bin to PATH)